### PR TITLE
Adding intermittency function (gamma_BC) to the Tecplot output

### PR DESCRIPTION
**Pull request by sametcaka**
Dear all,
This pull request is to add the gamma_BC variable (the intermittency function of B-C transition model) to the output file for visualization purposes, so that the user can see the laminar and turbulent regions of the flow.
This does not work as is now, and needs to be corrected. Tim is already helping me on this, and I am trying to figure everything out according to his comments. Any additional advice/help is appreciated. Hope this would be ready for the next release.

### CODE REVIEW

**Comment by talbring** [comment]
There were a lot of unnecessary whitespace changes (make sure to set up the editor correctly so that it does not remove empty lines when you save). I cleaned it up.

**Comment by economon** [comment]
Samet, Tim: adding extra output sounds great! I am still seeing a lot of white space changes though, even after Tim's commit. Is there anything we can do about the rest of these?

**Comment by sametcaka** [comment]
Tom, I am not sure why those whitespaces are there, maybe because of the text editor I used. I will try to correct them. Also, there were some conflicts after I created my pull request, and I resolved them using the web ui. Maybe that would also be the source of the whitespaces you mentioned.

**Comment by talbring** [comment]
Should be solved now. I was not using git checkout to revert the files ... instead I copied the content of the files using meld. Apparently that's not a good idea :)  Samet, if necessary, I can also add the missing things today. Just let me know.

**Comment by sametcaka** [comment]
Tim, if you could do that, that will of course be great! I am still trying to get it work, but this might take time. If you add the missing things, I can follow your work, and understand it better.

**Comment by talbring** [comment]
Samet I finished your implementation. Let me know if it works for you.

**Comment by sametcaka** [comment]
Tim, it works perfectly. This was a very nice "tutorial" for me, indeed  :) Thank you very much again, Tim.
 
If everything else is OK, it would be merged.
Cheers!

--Samet

File: SU2_CFD/include/variable_structure.hpp [hunk 1]
```diff
@@ -3638,6 +3649,10 @@ class CTurbVariable : public CVariable {
  */
 
 class CTurbSAVariable : public CTurbVariable {
+
+private:
+  su2double gamma_BC; /*!< \brief Value of the intermittency for the BC trans. model. */
```
Comment (economon) [inline SU2_CFD/include/variable_structure.hpp]: Any chance you will extend the approach to other turbulence models like SST? If so, we could move the gamma_BC data and getter/setter methods to the parent class. If not, this is fine.

Comment (sametcaka) [inline SU2_CFD/include/variable_structure.hpp]: I am so busy with my work and phd studies, and thus cannot find time to work on coupling my model with SST. So, we may leave it as is now, if thats fine.

Comment (talbring) [inline SU2_CFD/include/variable_structure.hpp]: I am gonna add an error message if you use the model with SST.

**Comment by talbring** [comment]
From my side it is ready to be merged.

**Comment by fpalacios** [comment]
No worries about the Travis failure, it is because of the changes in the metadata structure in restart files. We can merge this implementation.

**Comment by fpalacios** [comment]
Thanks @sametcaka for this implementation. The open-source contributions are the heart of SU2 and we need you all to push this project to the next level.

We are looking forward to see more improvements from you in the future! 

Best,
Francisco

**Pull request merged**